docs/getting-started/cases-req.asciidoc

```diff
@@ -13,15 +13,7 @@ You can create roles and define feature privileges at different levels to manage
 NOTE: To send cases to external systems, you need the
 https://www.elastic.co/subscriptions[appropriate license].
 
-When setting up feature access to cases, you can configure the following:
-
-- Base privileges for the {es-sec} solution
-- Sub-feature privileges for the Cases feature (toggle *Customize sub-feature privileges* to use this optional setting)
-- Privileges for actions and connectors
-
-NOTE: Privileges set at the sub-feature level will override base privileges for the {es-sec} solution. In other words, the Cases feature takes on the base privileges of the {es-sec} solution unless configured otherwise. For example, a role with `Read` base privileges only can view {es-sec} features and artifacts (including cases) within the {es-sec} app  but can't modify them. However, setting the role's sub-feature privileges for Cases to `All` allows the role to view and manage cases only.
-
-More information about the minimum privileges needed to use cases is below:
+To grant access to cases, set the {kib} space privileges for the *Cases* and *Actions and Connectors* features as follows:
```

Comment:
do we need to say that `Actions and Connectors` privileges are just here if you want to push a case to an external service like `serviceNow`?